test/CMakeLists.txt

```diff
@@ -10,17 +10,18 @@
 cmake_minimum_required(VERSION 3.1)
 
 find_package(doctest            REQUIRED)
-find_package(Threads)
 
 if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
-    project(xtensor-test)
+    project(xtensor-test C CXX)
```

Comment:
I'm not entirely against this, but:
* I've never used it, does it change anything?
* I don't think `C` is relevant, everything is C++